test/1.4/misc/T_notify_state.dml

```diff
@@ -13,17 +13,32 @@ header %{
     void state_cb(conf_object_t *obj, conf_object_t *_, lang_void *__) {
         (*get_count_pointer(obj))++;
     }
+    extern void exported_method(conf_object_t *);
+    void exported_trampoline(conf_object_t * obj, conf_object_t *_,
+                            lang_void *__) {
+        exported_method(obj);
+    }
+    extern void independent_exported_method(conf_object_t *);
+    void independent_exported_trampoline(conf_object_t * obj, conf_object_t *_,
+                                        lang_void *__) {
+        independent_exported_method(obj);
+    }
     void register_callbacks(conf_object_t *obj) {
         SIM_add_notifier(obj,
                          Sim_Notify_State_Change,
                          obj,
                          state_cb,
                          NULL);
-    }
+        notifier_type_t notifier_type = SIM_notifier_type("exported_entry");
```

Comment:
there is a `kebab-case` naming convention for notifier types.

Reply:
Although hell, why not.